_posts/2024-08-16-AI-chatting.md

```diff
@@ -0,0 +1,93 @@
+
+Let's try it out on a paper "Cooper, K., Donovan, J., Waterhouse, A., & Williamson, G. (2007). Cocoa and health: a decade of research. *British Journal of Nutrition*, 99(1), 1–11. doi:[10.1017/s0007114507795296](https://doi.org/10.1017/s0007114507795296)"
+
+Let's ask about the chocolate.
+
+![AI first question and answer](../img/AiQuestion1.png)
+
+Correct! The authors talk a lot about chocolate components, and how it can be a powerfull antioxidant.
+Wow! AI has given so much infromation, and it's true: chocolate really contains polyphenols and fatty acids.
+
+I was wondering, how much chocolate should a human eat in a day? Let's dive in this topic.
+
+![AI second question and answer](../img/AiQuestion2.png)
+
+*Only 40 grams?* Well, our health is our wealth. And AI helped us to uncover this information!
+
+## How does this work?
+
+In the background, JabRef analyses the linked PDF files of library entries. The information used after the indexing is then supplied to the AI, which, to be precise, in our case is a Large Language Model (LLM).
+The LLM is currently not stored on your computer. Instead, we have many integrations with AI providers (OpenAI, Mistral AI, Hugging Face), so you can choose the one you like the most.
+We send chunks of text to AI service and then receive processed responses. In order to use it you need to configure JabRef to use your API key.
+
+## Which AI provider should I use?
+
+For now, we recomend you chosing [OpenAI](https://platform.openai.com/docs/models).
+
+For Mistral AI you might need to make a subscription, while for OpenAI you can send money one time.
+
+Hugging Face gives you access to numerous count of models for free.
+However, it will take a very long time for Hugging Face to find a free computer resources for you, and the response time will be also long.
+
+In order to use any service, you always need an API key.
+Please head to the [AI user documentation](https://docs.jabref.org/ai/ai-providers-and-api-keys) to learn about how to receive a key and where to enter it in the preferences.
+
+## AI preferences
+
+Here are some new options in the JabRef preferences.
+
+![AI preferences](../img/AiPreferences.png)
+
+- "Enable AI functionality in JabRef": by default it is turned off, so you need to check this option, if you want to use the new AI features
+- "AI provider": you can choose either OpenAI, Mistral AI, or Hugging Face
+- "Chat model": choose the model you like (for OpenAI we recommend `gpt-4o-mini`, as to date, it is the cheapest and fastest, although we also recommend to look up the prices from time to time, as they are subject to change)
+- "API token": here you write your API token
+- "Expert settings": here you can change the parameters that affect how AI will generate your answers. If you don't understand the meaning of those settings, don't worry! We have experimented a lot and found the best parameters for you! But if you are curious, then you can refer to [user documentation]()
```

Comment:
Can you please rephrase "If you don't understand ... best parameters for you!" to a more professional sounding wording? Maybe: "The defaults provide good results. In case you would like to modify them, our [user documentation](https://docs.jabref.org/ai/preferences#ai-expert-settings) provides an explanation of it"

(This also adds the link to the user documentation. Currently, the link is empty.)